staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -44,6 +45,15 @@ const (
 // ServerRunOptions contains the options while running a generic api server.
 type ServerRunOptions struct {
 	AdvertiseAddress net.IP
+	// PeerCAFile is the ca bundle used by this kube-apiserver to verify peer apiservers'
+	// serving certs when routing a request to the peer in the case the request can not be served
+	// locally due to version skew.
+	PeerCAFile string
+
+	// PeerBindAddress is the IP for this kube-apiserver which is used by peer apiservers to route a request
+	// to this apiserver. This happens in cases where the peer is not able to serve the request due to
+	// version skew.
+	PeerBindAddress utilpeerproxy.PeerBindAddress
```

Comment:
validate this is not set if the UVIP feature is not enabled

Reply:
once the UVIP feature gate graduates to beta and enables by default, what happens if this field is unset? is there a default?

Reply:
Currently we pick the [publicAddress](https://github.com/kubernetes/kubernetes/pull/117740/files#diff-05cd394e18da9e91dbf8b3d82e1e0813be670ee37db43ca5af6f91a39f78d08fR500)  (which is the advertise-address or the bind-address) if peer-advertise-address is unavailable. Will need to set default to this in beta.